packages/cozy-harvest-lib/src/components/AccountForm.jsx

```diff
@@ -42,7 +44,10 @@ export class AccountField extends PureComponent {
       disabled: !isEditable,
       fullwidth: true,
       label: t(`fields.${localeKey}.label`, { _: name }),
-      placeholder: placeholder || t(`fields.${name}.placeholder`, { _: '' }),
+      placeholder:
+        getPlaceholderForEncryptedFields(this.props) ||
+        placeholder ||
+        t(`fields.${name}.placeholder`, { _: '' }),
```

Comment:
What about using only one method `getFieldPlaceholder(this.props)`?

Reply:
That was my first thought, let me give it another try.

Reply:
@CPatchane is that better ?

Reply:
Yep 👍

Reply:
@CPatchane and this ?

Reply:
Much better 👍